lightning/src/routing/gossip.rs

```diff
@@ -2311,6 +2325,15 @@ pub(crate) mod tests {
 			Ok(_) => panic!(),
 			Err(e) => assert_eq!(e.err, "Channel announcement node had a channel with itself")
 		};
+
+		// Test that channel announcements with the wrong chain hash are ignored.
```

Comment:
nit: Could we just include why it's the wrong chain hash here? I got a little confused with it in isolation but remembered we were dealing with testnet in these gossip tests.

"testnet vs mainnet" or something similar.